<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_habit_cell</class>
 <widget class="QWidget" name="add_habit_cell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="habit_name">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter Habit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="habit_frequency"/>
     </item>
     <item>
      <widget class="QPushButton" name="delete_habit_button">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
